sycl/tools/sycl-ls/sycl-ls.cpp

```diff
@@ -83,6 +116,23 @@ static void printDeviceInfo(const device &Device, bool Verbose,
     std::cout << Prepend << "Vendor     : " << DeviceVendor << std::endl;
     std::cout << Prepend << "Driver     : " << DeviceDriverVersion << std::endl;
 
+    // Get and print device UUID, if it is available.
+    if (Device.has(aspect::ext_intel_device_info_uuid)) {
+      auto UUID = Device.get_info<sycl::ext::intel::info::device::uuid>();
+      std::cout << Prepend << "UUID       : ";
+      for (int i = 0; i < 16; i++) {
```

Comment:
Should you be using ZE_MAX_DEVICE_UUID_SIZE instead of assuming size is 16

Reply:
That can be done. But, Isn't the UUID size fixed to 16: https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/supported/sycl_ext_intel_device_info.md#device-uuid?

Reply:
If it is documented as 16 then ok.

Reply:
Or just use a range-based for loop on UUID